Foundation.xcodeproj/project.pbxproj

```diff
@@ -2763,7 +2775,6 @@
 					"-DCF_CHARACTERSET_UNICODE_DATA_B=\\\\\"CoreFoundation/CharacterSets/CFUnicodeData-B.mapping\\\\\"",
 					"-DCF_CHARACTERSET_UNICHAR_DB=\\\\\"CoreFoundation/CharacterSets/CFUniCharPropertyDatabase.data\\\\\"",
 					"-DCF_CHARACTERSET_BITMAP=\\\\\"CoreFoundation/CharacterSets/CFCharacterSetBitmaps.bitmap\\\\\"",
-					"-Wno-nullability-completeness-on-arrays",
```

Comment:
Was this removed by the upgrade process? Is it implied by default now?

Reply:
No, I removed it manually. It was added in https://github.com/apple/swift-corelibs-foundation/pull/988 but 
a) I think its a useful warning to have,
b) The `build.ninja` doesnt have this flag so these warnings currently show up on Linux anyway and so this keeps the two builds in sync.

Reply:
OK, makes sense.